tests/Spout/Writer/XLSX/SheetTest.php

```diff
@@ -76,6 +77,18 @@ public function testSetSheetNameShouldThrowWhenNameIsAlreadyUsed()
     }
 
     /**
+     * @return void
+     */
+    public function testSetSheetVisibilityShouldCreateSheetHidden()
+    {
+        $fileName = 'test_set_visibility_should_create_sheet_hidden.xlsx';
+        // $customSheetName = 'CustomName';
+        $this->writeDataAndReturnSheetHidden($fileName);
+
+        $this->assertSheetVisibilityEquals(false,  $fileName, "The sheet visibility should have been changed to 'false'");
+    }
+
+    /**
```

Comment:
just below, can you rename the function `writeDataAndReturnSheetWithCustomName` into `writeDataToSheetWithCustomName` and change the `@return Sheet` to `@return void` please ?